lib/prisma.ts

```
@@ -0,0 +1,20 @@
+import { env } from '@/env';
+
+neonConfig.webSocketConstructor = ws;
+
+declare global {
+  var prisma: PrismaClient | undefined
+}
+
+const connectionString = env.DATABASE_URL;
+
+const adapter = new PrismaNeon({ connectionString });
+export const prisma = global.prisma || new PrismaClient({ adapter });
```

Biome (1.9.4)

[error] Shouldn't redeclare 'prisma'. Consider to delete it or rename it.

'prisma' is defined here:


(lint/suspicious/noRedeclare)